cdap-gateway/src/test/java/co/cask/cdap/gateway/handlers/StreamHandlerTestRun.java

```diff
@@ -137,9 +147,103 @@ public void testStreamInfo() throws Exception {
     urlConn = openURL(String.format("http://%s:%d/v2/streams/stream_info/info", HOSTNAME, port),
                       HttpMethod.GET);
     Assert.assertEquals(HttpResponseStatus.OK.getCode(), urlConn.getResponseCode());
-    StreamProperties properties = GSON.fromJson(new String(ByteStreams.toByteArray(urlConn.getInputStream()),
+    StreamProperties actual = GSON.fromJson(new String(ByteStreams.toByteArray(urlConn.getInputStream()),
                                                         Charsets.UTF_8), StreamProperties.class);
-    Assert.assertEquals(2, properties.getTTL());
+    urlConn.disconnect();
+    Assert.assertEquals(streamProperties, actual);
+  }
+
+  @Test
+  public void testPutStreamConfigDefaults() throws Exception {
+    int port = GatewayTestBase.getPort();
+
+    // Now, create the new stream.
+    HttpURLConnection urlConn = openURL(String.format("http://%s:%d/v2/streams/stream_info",
```

Comment:
I don't see anything that would clean up the streams between test methods.  Shouldn't the stream name here be unique ("stream_config_defaults" or something)?